doc/migrations/v0.46-v1.0.0.md

```diff
@@ -63,3 +64,8 @@ The following metrics were renamed:
 
 `libp2p_dialler_pending_dials` => `libp2p_dial_queue_pending_dials`
 `libp2p_dialler_in_progress_dials` => `libp2p_dial_queue_in_progress_dials`
+
+
+## Connection Manager
+
+We now only dial one peer at a time as this is a more efficient use of resources.  This means that the `maxParallelDialsPerPeer` option has been removed.
```

Comment:
I don't think this is right. We're removing parallel dials per peer, but not across peers.

Reply:
I've updated the note, hopefully it's clearer now.